admin/sql/create_tables.sql

```diff
@@ -50,6 +50,14 @@ CREATE TABLE follow_list (
 );
 ALTER TABLE follow_list ADD CONSTRAINT follow_list_name_creator_key UNIQUE (name, creator);
 
+CREATE TABLE missing_releases_musicbrainz (
```

Comment:
I think we should structure the table like this: https://gist.github.com/mayhem/9c278d7c9395008961e3cd2ee12d228c

As discussed in IRC.